train.py

```diff
@@ -54,6 +59,45 @@ def get_arguments():
     return parser.parse_args()
 
 
+def save(saver, sess, logdir, step):
+    model_name = 'model.ckpt'
+    checkpoint_path = os.path.join(logdir, model_name)
+    print('Storing checkpoint to {} ...'.format(checkpoint_path), end="")
+
+    if not os.path.exists(logdir):
+        os.makedirs(logdir)
+
+    saver.save(sess, checkpoint_path, global_step=step)
+    print(' Done.')
+
+
+def load(saver, sess, logdir):
+    print("Trying to restore saved checkpoints from {} ...".format(logdir), end="")
+
+    ckpt = tf.train.get_checkpoint_state(logdir)
+    if ckpt:
+        print("")
+        print("  Checkpoint found: {}".format(ckpt.model_checkpoint_path))
+        global_step = int(ckpt.model_checkpoint_path.split('/')[-1].split('-')[-1])
+        print("  Global step was: {}".format(global_step))
+        print("  Restoring...", end="")
+        saver.restore(sess, ckpt.model_checkpoint_path)
+        print(" Done.")
+        return global_step
+    else:
+        print(" No checkpoint found.")
+        return None
+
+
+def get_default_logdir():
```

Comment:
Maybe it would be simpler to just call
```python
datestring = "{0:%Y-%m-%dT%H-%M-%S}".format(datetime.now())
default_logdir = os.path.join(LOGDIR, 'train', datestring)
```
somewhere close to the top of the script.
What do you think?